package.json

```diff
@@ -7,17 +7,20 @@
     "clean": "rm -rf dist/ && rm -rf generated/",
     "start": "webpack-dev-server --config utils/build/webpack.dev.js",
     "build": "webpack --config utils/build/webpack.prod.js",
+    "server": "",
+    "test": "jest --collectCoverage",
```

Comment:
Like the other scripts, can test call 'sub' scripts - ie 'test:unit' for jest, 'test:e2e' for cypress etc?